src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -520,26 +520,25 @@ public static ColInfo Create(string name, PrimitiveType itemType, Segment[] segs
             }
         }
 
-        private sealed class Bindings : ISchema
+        private sealed class Bindings
         {
+            /// <summary>
+            /// <see cref="Infos"/>[i] stores the i-th column's name type loaded from the input text file.
```

Comment:
>name type [](start = 64, length = 9)

Not a big deal, but what is a "name type"?

Reply:
`name type` is changed to `name and type`. Thanks.

---
In reply to: [247681984](https://github.com/dotnet/machinelearning/pull/2140#discussion_r247681984) [](ancestors = 247681984)